src/main/client/_macros.ftl

```diff
@@ -34,7 +34,7 @@
       [#case "KeyAlgorithm"][#return "KeyAlgorithm?"/]
       [#default]
         [#if type?starts_with("Collection")]
-          [#return type?replace("Collection", "List")?replace("UUID", "string")/]
+          [#return type?replace("Collection", "List")?replace("UUID", "string")?replace("String", "string")/]
```

Comment:
In .NET the collection generic is `string` not `String`. 1st time we've had to do this in a URL param.